### PR TITLE
Exception handling

### PR DESCRIPTION
**Pull request by n0rdy**

### CODE REVIEW

File: registry-manager-rest-server/src/main/java/de/adorsys/registry/manager/exception/ExceptionHandlingAdvisor.java [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+import org.springframework.web.bind.annotation.ExceptionHandler;
+
+import java.io.UncheckedIOException;
+import java.util.List;
+
+@ControllerAdvice
+public class ExceptionHandlingAdvisor {
+    private static final Logger logger = LoggerFactory.getLogger(ExceptionHandlingAdvisor.class);
+
+    @ExceptionHandler(IbanException.class)
```
Comment (spetrychenko) [inline registry-manager-rest-server/src/main/java/de/adorsys/registry/manager/exception/ExceptionHandlingAdvisor.java:22]: I think `409` or `400` can be returned, when the user provides the wrong data we reply with `4xx` statuses.
What do you think?

File: registry-manager-rest-server/src/main/java/de/adorsys/registry/manager/exception/ExceptionHandlingAdvisor.java [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+        String errorMessage = "Exception during the IBAN processing: IBAN is incorrect";
+        return ResponseEntity
+                       .status(HttpStatus.INTERNAL_SERVER_ERROR)
+                       .body(new ErrorResponse(List.of(errorMessage)));
+    }
+
+    @ExceptionHandler(UncheckedIOException.class)
+    public ResponseEntity<ErrorResponse> handle(UncheckedIOException ex) {
+        logError(ex);
+        String errorMessage = "Exception during the IO process";
```
Comment (spetrychenko) [inline registry-manager-rest-server/src/main/java/de/adorsys/registry/manager/exception/ExceptionHandlingAdvisor.java]: this part of code could be extracted into separate method and re-used for errors

File: registry-manager-rest-server/src/main/java/de/adorsys/registry/manager/exception/ExceptionHandlingAdvisor.java (hunk 1 above)
Comment (n0rdy) [reply registry-manager-rest-server/src/main/java/de/adorsys/registry/manager/exception/ExceptionHandlingAdvisor.java:22]: yeah, you are right, fixed, thanks

File: registry-manager-rest-server/src/main/java/de/adorsys/registry/manager/exception/ExceptionHandlingAdvisor.java (hunk 2 above)
Comment (n0rdy) [reply registry-manager-rest-server/src/main/java/de/adorsys/registry/manager/exception/ExceptionHandlingAdvisor.java]: fixed, thanks

**Review by spetrychenko — APPROVED** [review]

**Pull request merged**